Fix querying sql (#14026)
apache/druid: docs/tutorials/tutorial-query.md

9. Finally, click  `...`  and **Edit context** to see how you can add additional parameters controlling the execution of the query execution. In the field, enter query context options as JSON key-value pairs, as described in [Context flags](../querying/query-context.md).  

That's it! We've built a simple query using some of the query builder features built into the web console. The following
sections provide a few more example queries you can try.

See [Query SQL over HTTP](#query-sql-over-http) for an example of how to use the Druid SQL HTTP API. 

## More Druid SQL examples

> Try the following queries to learn a few more Duid SQL tricks:

Reviewer: ```suggestion
Try the following queries to learn a few more Druid SQL tricks:
```

```sql
SELECT channel, page, SUM(added)
FROM wikipedia WHERE TIME_IN_INTERVAL("__time", '2016-06-27/2016-06-28')
GROUP BY channel, page
ORDER BY SUM(added) DESC
```

![Query example](../assets/tutorial-query-07.png "Query example")

## Query SQL over HTTP

> You can submit native queries [over HTTP](../querying/sql-api.md#submit-a-query). The request body should be a JSON object, with the value for the key `query` containing text of the query:

Reviewer: ```suggestion
You can submit native queries [over HTTP](../querying/sql-api.md#submit-a-query). The request body is a JSON object in which the `query` field contains the text of the query:
```
